COPYRIGHT.txt

```diff
@@ -1939,3 +1939,8 @@ License: Zlib
  2. Altered source versions must be plainly marked as such, and must not be
     misrepresented as being the original software.
  3. This notice may not be removed or altered from any source distribution.
+
+Files: ./thirdparty/nvapi/nvapi_minimal.h
+Comment: Stripped down version of "nvapi.h" from the NVIDIA NVAPI SDK
+Copyright: 2019-2022, NVIDIA CORPORATION. All rights reserved.
+License: MIT
```

Comment:
```suggestion
Copyright: 2019-2022, NVIDIA Corporation
License: Expat
```
(This file follows a format defined by Debian and they use "Expat" for the typical MIT license, to disambiguate from a few other licenses originating from MIT with slightly different languages.